EIPS/eip-5553.md

```diff
@@ -0,0 +1,260 @@
+The standard is kept very generic on purpose to allow the industry to evolve new ecosystems that can all rely on the same basic standard at their core.
+
+This standard allows market participants to:
+1) Observe the canonical on-chain representation of an intellectual work
+2) Discover its attached metadata
+3) Discover its related royalty rights holders
+4) In the future, this will enable building registration, licensing and payout mechanisms for intellectual property assets.
+
+The standard is based on an `EIP-721` based contract implementing the new `IWorksRegistration` interface.
+The `IWorksRegistration` interface contains:
+
+### royaltyInterestTokens() function
+returns an array of addresses related to [EIP-20](./eip-20.md) tokens that represent royalty interest allocation to different types of interested parties that expect future royalty distribution. These royalty tokens represent a more granular and streamlined way to declare royalty splits for multiple collaboration participants for the creation of the work. For example, for a musical work, we might have two tokens representing the Composition/Writing/Publishing royalty rights side, and the Recording/Master side. 
+
+These royalty tokens are distributed to the collaboration participants and can later be queried for the various holders, for purposes of distribution of royalties - i.e if you hold 10% of a royalty token, you might get 10% of financial distribution related to that type of royalty.
+
+### metadataURI() function
+returns the URI to a metadata file containing any required metadata for the work. Each work type will have its own metadata standard, defined separately, and is not the focus of this standard. The hope is that the file will be hosted in IPFS or arweave or other resource hash based access systems in which we know the file's contents are not changeable without changing the URI.
+
+### changeMetadataURI() function
+Allows changing the URI when updating the metadata for the work. Will also trigger the event `MetadataChanged`
+
+### ledger() function
+returns the address of the registry or registrar contract, or an EOA account that initialized the work and associated royalty tokens. A work might be registered in multiple places by different actors for different purposes, and this enables market participants to discover which registry mechanism is the parent of the work and might have special access rights to manage the work.
+
+## Motivation
+There is no accepted standard mechanism to license a work, or to represent it, except using traditional NFTs. But regular NFTs only represent a collectible item use case, and cannot easily represent more complicated use cases of licensing a work for different types of uses.
```

Comment:
[Peer review] Did you get a chance to check out EIP-5218? Do you think 5218's relevant to this EIP?

Reply:
I've seen it before  - but had not considered connecting the dots. I might tryu to mention here in this context as a possible use case along with works registration/